### ISSUE TITLE
Question on 3.3 Accessibility Requirement ERD Relationship Constraint of One (or Many)?

### ISSUE THREAD

**Issue by goodwitch** [w3c/wcag-act#122]
[3.3 Accessibility Requirements](https://www.w3.org/TR/act-rules-format/#structure-accessibility-requirements) - Is there an assumption that there will be a 1 to 1 relationship between an ACT Rule and an Accessibility Requirement? Or will you allow 1 ACT Rule to be mapped to 2 or more Accessibility Requirements (there are some failures that trigger more than 1 WCAG 2.0 SC).  For example - an image that is an active link with no alt text will trigger a failure of 1.1.1 and a failure of 2.4.4 Link Purpose (in context).

**Comment by WilcoFiers** [comment]
As defined in the document: An ACT Rule MAY be a complete or partial test for one or more accessibility requirements.
